modules/ROOT/pages/update-acm.adoc

```diff
@@ -49,22 +49,34 @@ Attempting to upgrade the package without this feature enabled causes the proces
 .. Continue with normal package installation.
 . Select *Install for admins only*.
 . Select *Upgrade*.
-
++
 This starts the installation process.
-
++
 When the upgrade finishes, you receive a notification email.
-
-If you are upgrading from package version 1.62 or before, after installing, ensure both `Member` and `Guest` profiles have field-level `Read` access to the `ParentId` field in the `CommunityAPI` object:
-
-. Configure the `Member` profile:
-.. Click *Setup* > *Profiles*.
-.. Open the `Member` profile.
-.. Click *Object settings* > *CommunityAPI*.
-.. Ensure the `ParentId` field has `Read` access.
-.. Save.
-. Configure the `Guest` profile:
-.. Click *Community builder* > *Settings* > *General*.
-.. Open the `Guest` profile.
-.. Click *Object settings* > *CommunityAPI*.
-.. Ensure the `ParentId` field has `Read` access.
-.. Save.
+. If you are upgrading from package version 1.62 or before, after installing, ensure both `Member` and `Guest` profiles have field-level `Read` access to the `ParentId` field in the `CommunityAPI` object:
```

Comment:
1.62 or before > 1.62 or *earlier*, per style guide to refer to versions

Reply:
Thank you! Fixed.